views/pages/categoryRadar.ejs

```diff
@@ -40,38 +40,73 @@
     </div>
 
     <div class="row" style="padding-top: 20px;">
-        <div class="col-sm-6">
-            <ul class="list-group">
-                <%
-                var heading = "";
+        <div class="row top-buffer">
+            <div class="col-sm-6">
 
+                <%
                 if( technologies.length == 0) {
                 %>
-
-                <li class="list-group-item active">No technologies found</li>
-
+                <p>No technologies found</p>
                 <%
-                }
-
-                for (var i = 0, len = technologies.length; i < len; i++) {
-                    var technology = technologies[i];
-
-                if( technology.status != heading) {
-                    heading = technology.status;
+                } else {
+                    var heading = "";
                 %>
-                <li class="list-group-item active"><%= heading %></li>
-
-                <% } %>
-
-                <li class="list-group-item" id="tech<%=technology.num%>">
-                    <span class="badge"><%= technology.num %></span>
-                    <a href="/technology/<%= technology.id %>"><%= technology.name %></a>
-                </li>
-
+                <ul class="nav nav-pills" data-tabs="tabs">
+
+                    <%
+                    for (var i = 0, len = technologies.length; i < len; i++) {
+                        var technology = technologies[ i ];
+                    if( technology.status != heading) {
+                        heading = technology.status;
+                    %>
+                    <li class="<%- (i === 0 ) ? 'active' : '' %>">
+                        <a href="#<%= technology.status %>" data-toggle="tab"><%= technology.status %></a>
+                    </li>
+                    <%
+                    }
+                    }
+                    %>
+                </ul>
+
+                <div class="tab-content">
+                    <%
+                    var heading = "";
+                    var lastTechId = -1;
+                    var lastTechNum=-1;
+
+                    for (var i = 0, len = technologies.length; i < len; i++) {
+                        var technology = technologies[ i ];
+                    if( technology.status != heading) {
+                        heading = technology.status;
+
+                    if(i != 0) { %>
```

Comment:
Very convoluted logic, is it possible to simplify/comment on some of the code here?